app/controllers/account/users_controller.rb

```diff
@@ -64,11 +65,23 @@ def resource_params
     params.require(:user).permit(:first_name, :last_name, :email, :role)
   end
 
-  def collection
+  def admin_collection
     current_tenant.users.by_date
   end
 
-  def resource
-    collection.find(params[:id])
+  def admin_resource
+    admin_collection.find(params[:id])
+  end
+
+  def super_admin_collection
+    User.all.by_date
+  end
+
+  def super_admin_resource
+    super_admin_collection.find(params[:id])
+  end
+
+  def determine_resource
```

Comment:
def resource
  collection.find(params[:id])

and do change with collection requested above